examples/llm/src/models/mosaic_gpt.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, cfg: DictConfig):
         self.transformer.update(
             {'ln_f': nn.LayerNorm(cfg.d_model, device=cfg.init_device)})
 
+        # enables scaling output logits; similar to a softmax "temperature"
+        # PaLM paper uses scale 1/sqrt(cfg.d_model)
+        if cfg.get('logit_scale'):
+            self.register_buffer('logit_scale', torch.Tensor([cfg.logit_scale]))
+        else:
+            self.register_buffer('logit_scale', None)
+
```

Comment:
From your comment I guess this is in order to save `logit_scale` in the model checkpoint, but it could also just live as part of the `model_cfg` and then we would just use a float rather than a Tensor?

Thinking about this in relation to `alibi_bias` or `softmax_scale` or other args like that, should we just keep it as a `cfg` float for now? I agree we need a better solution for saving model configs colocated with the checkpoint weights, but not sure if we should do a mix of solutions.

Reply:
ok I can leave it as just part of `self.cfg`. 
I was just trying to be thorough.